src/ui/Facet/FacetSearchParameters.ts

```diff
@@ -64,7 +64,7 @@ export class FacetSearchParameters {
     var request: IGroupByRequest = {
       allowedValues: typedByUser.concat(this.alwaysInclude).concat(this.alwaysExclude),
       maximumNumberOfValues: nbResults,
-      completeFacetWithStandardValues: this.facet.options.lookupField ? false : true, // See : https://coveord.atlassian.net/browse/JSUI-728
+      completeFacetWithStandardValues: this.facet.options.lookupField || this.facet.options.allowedValues  ? false : true, // See : https://coveord.atlassian.net/browse/JSUI-728
```

Comment:
If lookupfield is specified, this would output something like : `completeFacetWithStandardValues : @lookupfield`, which is not what we want.

Should be instead something like : `this.facet.options.lookupfield == null ? this.facet.options.allowedValues == null ? true : false : false`

And then the ternary becomes insanely hard to read :)

So I would change it to something easier to read.

```
let completeFacetWithStandardValues = true;
if(this.facet.options.lookupField != null) {
  completeFacetWithStandardValues = false;
}
if(this.facet.options.allowedValues != null) {
  completeFacetWithStandardValues = false;
}

var request : IGroupByRequest = { [...] }

```